app/src/internal/java/com/kickstarter/ui/viewholders/FeatureFlagViewHolder.kt

```diff
@@ -1,35 +1,55 @@
 package com.kickstarter.ui.viewholders
 
-import android.view.View
-import androidx.core.content.ContextCompat
+import com.kickstarter.databinding.ItemFeatureFlagBinding
 import com.kickstarter.libs.rx.transformers.Transformers.observeForUI
+import com.kickstarter.model.FeatureFlagsModel
 import com.kickstarter.viewmodels.FeatureFlagViewHolderViewModel
-import kotlinx.android.synthetic.internal.item_feature_flag.view.*
 
-class FeatureFlagViewHolder(val view: View) : KSViewHolder(view) {
+class FeatureFlagViewHolder(val binding: ItemFeatureFlagBinding, val delegate: Delegate) : KSViewHolder(binding.root) {
 
     private val vm: FeatureFlagViewHolderViewModel.ViewModel = FeatureFlagViewHolderViewModel.ViewModel(environment())
 
+    interface Delegate {
+        fun featureOptionToggle(featureName: String, isEnabled: Boolean)
```

Comment:
Can you document this function? Also should this Delegate be inside the class or on the outside?

Reply:
it belongs internally to the ViewHolder